### ISSUE TITLE
[Meta]: Use allcontributors?

### ISSUE THREAD

**Issue by osfanbuff63** [Fabulously-Optimized/vanilla-installer#23]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

Since we're crediting all the contributors in the [README](https://github.com/Fabulously-Optimized/vanilla-installer#contributors), it might be worth using [allcontributors](https://allcontributors.org). Sorry for the ping, but since I believe it would need to be installed for the org (but you can give it access to only this repo), we'd need @Madis0 here.

In my opinion, it just pretty easily automates crediting everyone who contributed, whether in code or not.

For the record, [read this](https://allcontributors.org/docs/en/repository-maintainers) if you're concerned about maintaining with it, or not lol.

**Comment by Madis0** [comment]
What does that site do that https://github.com/Fabulously-Optimized/vanilla-installer/graphs/contributors doesn't?

**Comment by osfanbuff63** [comment]
> What does that site do that https://github.com/Fabulously-Optimized/vanilla-installer/graphs/contributors doesn't?

Mostly, recognizing people that contribute in ways that aren't code. For example, even creating a useful bug report should be considered a contribution, as should a security report, financial contributions (that example isn't relevant here, but it's to make the point) or even making a tutorial on how to use this. (That last example wasn't great, but it was what I thought of looking at the [emoji key](https://allcontributors.org/docs/en/emoji-key) lol)

tl;dr it credits people contributing in other ways, not just direct commits (or more specifically, code).

**Comment by Madis0** [comment]
Okay, the tool is installed. **You** will be the one who will set it up and keep it up to date. :grin:

**Comment by osfanbuff63** [comment]
Just as a warning to everyone, going to be using this as a setup ground if that's alright - feel free to unsubscribe esp when it pings for the bot lol

**Comment by osfanbuff63** [comment]
@all-contributors please add @osfanbuff63 for maintenance
@all-contributors please add @nsde for maintenance
@all-contributors please add @Kichura for infra
@all-contributors please add @RaptaG for code
@all-contributors please add @RozeFound for code

lets hope this works lol (again, sorry everyone for the ping)

**Comment by Kichura** [comment]
*(Out of character but: Hello!)*

**Comment by osfanbuff63** [comment]
@all-contributors please add @RaptaG for code
@all-contributors please add @RozeFound for code

(again, it failed last time for the two of you lol)

**Comment by RaptaG** [comment]
D:

**Comment by nsde** [comment]
> D:

Yesterday, I made a contributor widget from scratch ([proof](https://github.com/nsde/orbit)), which should display everyone, so don't worry about the @allcontributors bot not working.

![image](https://user-images.githubusercontent.com/67185896/196159207-452399cd-ed4a-442f-a38c-d986b571d5cc.png)

**Comment by osfanbuff63** [comment]
15 hours later lets see if it works

@all-contributors please add @RaptaG for code
@all-contributors please add @RozeFound for code

**Comment by osfanbuff63** [comment]
@all-contributors please add @RaptaG for code

**Comment by osfanbuff63** [comment]
ugh

**Comment by RaptaG** [comment]
F
